src/test/resources/features/Sale_CPQ_Forecasting.feature

```diff
@@ -2,10 +2,18 @@
 Feature: SalesForce_Forecasting
 
   @Web @salesforce @Forecast
-  Scenario: 2: Sale-CPQ-Forecasting
+  Scenario Outline: 2: Sale-CPQ-Forecasting
     Given navigate to salesforce
     And login to salesforce
     And navigate to setup screen
     And Check Forecast access for logged in User
+    Then Create Users Hierarchy for Forecast
     And Assign a Role to the Forecast User
     And Check users shown in the Forecasts Hierarchy
+    Then Check users shown in the Forecasts Hierarchy
```

Comment:
we have two same lines with different tags And and Then (line no 12 and 13) please check it.